### ISSUE TITLE
Conflating .NET and C# in documentation and marketing material

### ISSUE THREAD

**Issue by lilred** [OmniSharp/OmniSharp.github.io#7]
.NET is a software ecosystem and platform developed by Microsoft, with multiple languages running on top of it - primarily C#, F# and VB.NET.

Quoting the OmniSharp tagline:

>OmniSharp - Cross platform .NET development in the editor of your choice

The issue is that, at the time of writing, OmniSharp only provides tools for C# development. This is rather misleading for developers looking for F# or VB.NET tools. I'd appreciate if the documentation was updated to reflect that OmniSharp is currently a C#-only project.

The often-suggested alternative is to simply add F# and VB.NET support to OmniSharp. I can only talk for F# here, but there is already a sizeable initiative in the fsharp/fsharpbindings repo; that itch is already being scratched.

**Comment by nosami** [comment]
Hi!

I've reached out to the F# guys before - https://groups.google.com/forum/#!msg/fsharp-opensource/ca74UBG04v4/Y6NmZleUrHQJ It would be great if we could collaborate in some way or another. I've used F# in emacs myself, I realise fsharpbinding is an already established product with a thriving community. 

Pretty sure we'll have VB.Net support soon enough though... it should be simple enough to add. It doesn't require many changes to our existing code. F# on the other hand uses completely different tech - I'm not sure if it makes any sense to try and integrate into OmniSharp (especially as fsharpbinding is so good already)
